pkg/skaffold/build/jib/gradle.go

```diff
@@ -97,15 +99,22 @@ func getSyncMapCommandGradle(ctx context.Context, workspace string, a *latestV1.
 }
 
 // GenerateGradleBuildArgs generates the arguments to Gradle for building the project as an image.
-func GenerateGradleBuildArgs(task string, imageName string, a *latestV1.JibArtifact, skipTests, pushImages bool, deps []*latestV1.ArtifactDependency, r ArtifactResolver, insecureRegistries map[string]bool, showColors bool) []string {
-	args := gradleBuildArgsFunc(task, a, skipTests, showColors, MinimumJibGradleVersion)
+func GenerateGradleBuildArgs(task, imageName string, a *latestV1.JibArtifact, platforms platform.Matcher, skipTests, pushImages bool, deps []*latestV1.ArtifactDependency, r ArtifactResolver, insecureRegistries map[string]bool, showColors bool) []string {
+	minVersion := MinimumJibGradleVersion
+	if platforms.IsMultiPlatform() {
+		minVersion = MinimumJibGradleVersionForMultiPlatform
+	}
```

Comment:
Is cross platform builds ie. when `len(platforms)==1` is support by `MinimumJibGradleVersion` ?

Reply:
fixed